### PR TITLE
Add ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by renanborgez**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+## Actual Behavior
+
+
+## Steps to Reproduce the Problem
+
+    1.
+    1.
+    1.
```
Comment (dicearr) [inline .github/ISSUE_TEMPLATE.md]: 1.
2.
3.

**Comment by renanborgez** [comment]
updated

**Comment by renanborgez** [comment]
@dicearr maybe it is better -> https://github.com/wearereasonablepeople/webpacker/issues/templates/edit

**Comment by renanborgez** [comment]
closed in favor of #39

**Pull request closed without merging**